src/main/java/seedu/address/model/course/Course.java

```diff
@@ -2,34 +2,35 @@
 
 import static java.util.Objects.requireNonNull;
 
+import java.util.Objects;
+import java.util.function.Predicate;
+
 import javafx.collections.ObservableList;
 import javafx.collections.transformation.FilteredList;
 import seedu.address.commons.util.ToStringBuilder;
-import seedu.address.model.person.NameContainsKeywordsPredicate;
 import seedu.address.model.person.Student;
 import seedu.address.model.person.UniqueStudentList;
 
-import java.util.Objects;
+
 
 /**
  * Represents a Course in the course list.
  * Guarantees: details are present and not null, field values are validated, immutable.
  */
 public class Course {
 
+    public static final String MESSAGE_CONSTRAINTS =
+            "Course code should contain a two- or three-letter prefix, "
+                    + "four digits course code and an optional one-letter suffix";
```

Comment:
Good job on making the constraints clear so that user has an idea what type of course code is required.